templates/frontend/pages/indexJournal.tpl

```diff
@@ -28,6 +28,12 @@
 		</div>
 	{/if}
 
+    {* Journal Description *}
```

Comment:
We use tab indents. You should be able to update your editor to use tab indents and automatically format the code on save.

Reply:
I've got 4 tab indents set -- do you mind double-checking mine to make sure I got that one right? :) Sometimes it doesn't change or appear as tabs/spaces.